portal-kernel/src/com/liferay/portal/kernel/cookies/CookiesManager.java

```diff
@@ -27,6 +27,14 @@
 @ProviderType
 public interface CookiesManager {
 
+	public boolean addCookie(
+		Cookie cookie, HttpServletRequest httpServletRequest,
+		HttpServletResponse httpServletResponse);
+
+	public boolean addCookie(
+		Cookie cookie, HttpServletRequest httpServletRequest,
+		HttpServletResponse httpServletResponse, boolean secure);
+
 	public boolean addCookie(
 		int consentType, Cookie cookie, HttpServletRequest httpServletRequest,
 		HttpServletResponse httpServletResponse);
```

Comment:
Did we consider using an enum for `consentType`? To make the valid options more explicit.

Reply:
I guess they decided to use the `int` solution with the provided constants because they wanted to make it consistent with the frontend part.